controllers/user_controller.go

```diff
@@ -0,0 +1,65 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/drift-org/backend/models"
+	"github.com/gin-gonic/gin"
+	"github.com/kamva/mgm/v3"
+)
+
+type UserController interface {
+	Info(context *gin.Context)
+}
+
+type userController struct {
+}
+
+func NewUserController() UserController {
+	return &userController{}
+}
+
+func (ctrl *userController) Info(context *gin.Context) {
```

Comment:
So far, we have been naming our controllers after CRUD operations (we haven't done anything other than that). Your route is `GET localhost/user/info`, but IMO this is essentially a `GET localhost/user` route, (get user based on id). So, (imo) we should rename this controller to `Get` (similar to how we have `Create` in group_controller). Let's discuss at meeting.

Reply:
Agreed we can keep this as the default GET route for users. Switching it to `GET localhost/user`